x-pack/plugins/alerting/server/routes/lib/rewrite_rule.test.ts

```diff
@@ -43,8 +43,15 @@ const sampleRule: SanitizedRule<RuleTypeParams> & { activeSnoozes?: string[] } =
         notifyWhen: 'onThrottleInterval',
         throttle: '1m',
       },
-      // @ts-expect-error upgrade typescript v4.9.5
-      alertsFilter: { query: { kql: 'test:1', dsl: '{}', filters: [] } },
+      alertsFilter: { query: { kql: 'test:1', filters: [] } },
+    },
+  ],
+  systemActions: [
+    {
+      id: 'ccc',
+      actionTypeId: 'ddd',
```

Comment:
super nit: Do you mind if we add a more readable `actionTypeId` like `.test-system-action`?